example/example-app/src/App.tsx

```diff
@@ -1,10 +1,11 @@
 import React, { useEffect, useRef } from "react";
-import cgl, { ClustergrammerProps } from "./cg";
+import cgl from "./cg";
+import { ClustergrammerProps } from "./cg/index.types";
```

Comment:
```suggestion
import type { ClustergrammerProps } from "./cg/index.types";
```

Could be better?

Reply:
Yeah, it's definitely not necessary but is nice. I can do it in this instance now that it's coming from a pure types file, for sure :)